csfieldguide/templates/interactives/number-memory.html

```diff
@@ -17,13 +17,13 @@
       <button id="start-button" type="button" class="btn btn-success">{% trans 'Start' %}</button>
     </div>
     <div class="col-12">
-      <button id="submit-button" type="button" class="btn btn-primary d-none mb-1">{% trans 'Submit' %}</button>
+      <button id="submit-button" type="button" class="btn btn-secondary d-none mb-1">{% trans 'Submit' %}</button>
     </div>
     <div class="col-12">
-      <button id="reset-button" type="button" class="btn btn-danger d-none mb-1">{% trans 'Restart' %}</button>
+      <button id="reset-button" type="button" class="btn btn-primary d-none mb-1">{% trans 'Start over' %}</button>
     </div>
     <div class="col-12">
-      <button id="reset-hard-button" type="button" class="btn btn-danger d-none mb-1">{% trans 'Hard Mode' %}</button>
+      <button id="reset-hard-button" type="button" class="btn btn-success d-none mb-1">{% trans 'Continue' %}</button>
```

Comment:
Nit picky but can you please change the ID to something including the word 'continue' so it is easier to read/identify in the JS code.